src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
This doesn't work now, we got rid of skipping tests for specific versions on the infrastructure. We would skip TargetFrameworkMonikers.NetFramework when targetting any netfx version (which doesn't have the infra to run tests against them). We only support now running tests on netfx (which will build against 461, but will run against latest installed framework on running machine). So either we skip it with TargetFrameworkMonikers.NetFramework or add a `[ConditionalFact(nameof(Foo))]`

where  Foo:
```cs
public static bool Foo => !PlatformDetection.IsFullFramework || PlatformDetection.IsNet462OrNewer();
```